src/www/ui_components/app-root.js

```diff
@@ -343,10 +343,11 @@ export class AppRoot extends mixinBehaviors([AppLocalizeBehavior], PolymerElemen
               <iron-icon icon="outline-icons:outline" slot="item-icon"></iron-icon>
               <span class="item-label">[[localize('servers-menu-item')]]</span>
             </paper-icon-item>
-            <paper-icon-item name="feedback">
+            <!-- TODO(daniellacosse): fix sentry and restore feedback view -->
```

Comment:
In Outline we usually don't use `TODO(username)`.  I guess it's fine, but maybe `TODO(bug number)` would be better.

Reply:
I use TODO(username), but I agree that a bug is more useful.